morse_decode.rb

```diff
@@ -0,0 +1,13 @@
+def decode(morse)
+  morse_alphabet = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+    '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+    '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----.' => "'", ' ' => '',
+    '   ' => ' '
+  }
+  decoded_message = morse.split(/(\s{1,})/).map { |char| morse_alphabet[char] }.join
+  print decoded_message
+end
+
+decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ ] Based on the requirements you should have 3 methods:

  - The first one decodes one character.
  - The second one decodes a word using the first method
  - The third method decodes a sentence using the second method.

  Kindly split the implementation into three as outlined above.

- [ ] The methods should return the decoded message, unlike printing it as you have done on 10 here